CMakeLists.txt

```
@@ -18,6 +18,9 @@ if(NOT TTLS_NOTEST)
   add_subdirectory(3rdparty/googletest/googletest)
 
   set(CMAKE_CXX_CLANG_TIDY clang-tidy-10)
+
+  add_compile_options(-g -O0 -fsanitize=thread)
+  add_link_options(-fsanitize=thread)
```

```suggestion
  if(CMAKE_BUILD_TYPE STREQUAL Debug)
    add_compile_options(-fsanitize=thread)
    add_link_options(-fsanitize=thread)
  endif()
```
If one wants to make a release build for production, one should not need to enable TTLS_NOTEST.